proposals/csharp-13.0/overload-resolution-priority.md

```diff
@@ -278,7 +278,7 @@ There are a few more locations like [this](#application-error-or-warning-on-over
 * Lambdas - Similarly, lambdas are never subject to overload resolution, so the implementation blocks them. Should that be confirmed?
 * Destructors - again, currently blocked.
 * Static constructors - again, currently blocked.
-* Local functions - These are not currently blocked, because they _do_ undergo overload resolution, you just can't overload them. This is simlar to how we don't
+* Local functions - These are not currently blocked, because they _do_ undergo overload resolution, you just can't overload them. This is simular to how we don't
```

Comment:
```suggestion
* Local functions - These are not currently blocked, because they _do_ undergo overload resolution, you just can't overload them. This is similar to how we don't
```